benchmark_runner/workloads/workloads_operations.py

```diff
@@ -231,7 +231,7 @@ def _create_pod_run_artifacts(self, pod_name: str):
         :return: run results list of dicts
         """
         result_list = []
-        pod_log_file = self._create_pod_log(pod=pod_name)
+        pod_log_file = self._create_pod_log(pod=pod_name, type='.csv')
```

Comment:
Log files are not .csv.  The most common extension is .log, or perhaps .txt.

Reply:
I have updated it to be general log_type.
In addition, I passed the exact log type direct for specific workload.
For example in vdbench the workload results is in CSV format.